HealthSDK/GiniHealthSDKExample/GiniHealthSDKExample/de.lproj/Localizable.strings

```diff
@@ -14,3 +14,12 @@
 "giniHealthSDKExample.error.contains.multiple.invoices" = "Dokument enthält mehrere Rechnungen";
 "giniHealthSDKExample.error.invoice.not.payable" = "Rechnung ist nicht zahlbar";
 "gini.health.paymentcomponent.payment.info.gini.privacypolicy.link" = "https://www.google.com/"; // This should be overwrited by client with their privacy policy link
+"giniHealthSDKExample.order.detail.Alert.FieldError" = "Die Zahlungsdetails sind unvollständig";
```

Comment:
[Suggestion]: please rename the strings ginihealth.example.order.detail.alert.field.error